public/less/style.less

```diff
@@ -7,4 +7,8 @@
 
 .poll-view-result-avatar {
   width: 50px;
+}
+
+.poll-creator {
+  z-index: 11000 !important;
```

Comment:
> The topic composer has a z-index of 10000

This didn't use to be an issue, but I suppose psychobunny decided on an insane z-index for the new composer :smile: